### DISCUSSION TITLE
Vercel not showing files/folders in root

### DISCUSSION THREAD

**Discussion by BigAndyBoy** [vercel/community#6846] [Help]
### Summary

I am having a problem where for some reason Vercel can only see 1 file and 3 folders in the root when in reality there are much more. When I tried to print all of these, I got the following:
___vc
api
node_modules
package.json

when in reality, the root contains all of the following
.idea
about
api
fonts
images
node_modules
public
resourcepacks
file1.txt
file2.txt
file3.txt
file4.txt
file5.txt
index.html
package-lock.json
package.json
styles.css
vercel.json

Why is it not able to access these, especially the .txt files?

The root should have access to these files but for some reason cannot see them and I cannot run things that involve them, even though things like index.html run fine but are also not shown, as it runs visibly as an html website. What is the problem?

### Steps to Reproduce

I made a vercel based off of a github repo, and tried to run download.js in the api folder using the file1.txt, etc. When it would run what was in the download.js, it was unable to find these files.

**Comment by versecafe** [comment]
@BigAndyBoy the build output likely does not include the txt file, can you share your build outputs as well as checking if the files are being sent to Vercel at all by checking the deployment source files at `https://vercel.com/[team]/[project]/[deployment ID]/source`

<img width="1240" alt="image" src="https://github.com/vercel/community/assets/147033096/d8730bef-c451-4d59-8d80-40438e4ddcf9">

**Reply by BigAndyBoy** [reply]
This is my source output for package.json:
![image](https://github.com/vercel/community/assets/162652000/1b9f7840-e199-4938-8c16-70ec03dd9b67)

For my builds, I have this as my file setup for the root, and it surprisingly does show the file1.txt and everything similar, but not package.json.
![image](https://github.com/vercel/community/assets/162652000/aae81795-f342-4905-9166-d52ea1a6d74d)
